cartography/intel/aws/cloudfront.py

```diff
@@ -39,6 +39,16 @@ def get_cloudfront_distributions(boto3_session: boto3.session.Session) -> List[D
 def trtansform_distribution(dists: List[Dict]) -> List[Dict]:
     distributions = []
     for distribution in dists:
+        bucket_items = distribution.get("Origins", {}).get("Items", [])
+        bucket_names = []
+        for item in bucket_items:
+            bucket_id = item.get("Id", "")
+
+            if bucket_id.startswith("S3-"):
+                bucket_name = bucket_id[3:]
+                bucket_names.append(bucket_name)
+
+        distribution['S3bucketName'] = bucket_names
```

Comment:
property name needs to change. we can't call `S3BucketName` for an array of items. Please fix this.